app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,71 @@
+require './config/environment'
+class TweetsController < ApplicationController
+  get '/tweets' do
+    if is_logged_in?
+      @tweets = Tweet.all
+      erb :'/tweets/tweets'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/new' do
+    if is_logged_in?
+      erb :'/tweets/create_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  post '/tweets' do
+    if params[:content] == ""
+      redirect to "/tweets/new"
+    else
+      current_user.tweets.create(content: params[:content])
+      redirect '/tweets'
+    end
+  end
+
+  get '/tweets/:id' do
+    if is_logged_in?
+      #binding.pry
+      @tweet = Tweet.find_by_id(params[:id])
+      erb :'tweets/show_tweet'
+    else
+      redirect '/login'
+    end
+  end
+
+  get '/tweets/:id/edit' do
+    if is_logged_in?
+      @tweet = Tweet.find_by_id(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?